controllers/api.js

```diff
@@ -31,15 +31,15 @@ bitcoinclient.request('getblockchaininfo', false, function(err, info) {
   }
 });
 
-lightning.getInfo({}, function(err, info) {
+lightning.getInfo({}, function (err, info) {
   if (err) {
     console.error('lnd failure');
     console.dir(err);
     process.exit(3);
   }
   if (info) {
     console.info(info);
-    if (!info.synced_to_chain) {
+    if (!info.synced_to_chain && !info.syncedToChain) {
```

Comment:
where is this `syncedToChain` coming from? its not in the spec https://github.com/lightningnetwork/lnd/blob/daf7c8a85420fc67fffa18fa5f7d08c2040946e4/lnrpc/rpc.proto#L1560
https://api.lightning.community/#getinfo

Reply:
For me, it returned that on the latest LND

Reply:
@lukechilds can you explain that?

Reply:
If that information helps, everything was camelCase, not snake_case

Reply:
It's in the link you provided @Overtorment: https://github.com/lightningnetwork/lnd/blob/daf7c8a85420fc67fffa18fa5f7d08c2040946e4/lnrpc/rpc.proto#L1598

Reply:
@lukechilds But why is it camelCase? Shouldn't it be snake_case?

Reply:
Oh I'm not too sure.

Maybe the JS client lib that interfaces with LND maps all snake_case names to camelCase and exposes both since that's the convention in JavaScript?

Reply:
Yeah, that is probably the issue. That explains why it happened after the dependencie update